frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-config.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+import * as classNames from 'classnames';
+import {
+  InputGroup,
+  FormGroup,
+  TextInput,
+  Button,
+  ValidatedOptions,
+  Tooltip,
+} from '@patternfly/react-core';
+import { global_palette_blue_300 as blueInfoColor } from '@patternfly/react-tokens/dist/js/global_palette_blue_300';
+import { PencilAltIcon, EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import { setEncryptionDispatch, parseURL, kmsConfigValidation } from './utils';
+import { KMSConfigureProps } from './providers';
+import { advancedKMSModal } from '../modals/advanced-kms-modal/advanced-kms-modal';
+import { ActionType } from '../ocs-install/internal-mode/reducer';
+import { VaultConfig } from '../../types';
+import './kms-config.scss';
+
+export const ValutConfigure: React.FC<KMSConfigureProps> = ({
+  state,
+  dispatch,
+  className,
+  mode,
+  isWizardFlow,
+}) => {
+  const { t } = useTranslation();
+
+  const kms: VaultConfig = state.kms?.['vault'] || state.kms;
```

Comment:
will change it to `const kms: VaultConfig = state.kms?.['vault'];` once older OCS wizard code is removed from the repository.